tools/scripts/generate_hal_matrix.py

```diff
@@ -202,7 +203,11 @@ def hal_create_table(targets, platforms, common_table=False):
 {% endfor %}</tr><tr>
 <th align="left">Peripheral</th>
 {% for fam in families  -%}
+{% if fam[0] == "sam" -%}
+<th align="center">{{ fam[1] | upper | replace("X", "x") }}</th>
```

Comment:
```suggestion
<th align="center">{{ fam[1] | upper | replace("X", "x") | replace("/", "</br>") }}</th>
```
or something like that

Reply:
Good idea, thanks!